x-pack/plugins/aiops/public/components/change_point_detection/chapge_point_detection_page.tsx

```diff
@@ -0,0 +1,171 @@
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <SplitFieldSelector value={requestParams.splitField} onChange={setSplitField} />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      {isLoading ? (
+        <EuiProgress size="xs" color="accent" />
+      ) : (
+        <EuiHorizontalRule size="full" margin="s" />
+      )}
+
+      {annotations.length === 0 ? (
+        <EuiCallOut
+          size="s"
+          title={
+            <FormattedMessage
+              id="xpack.aiops.changePointDetection.noChangePointsFoundTitle"
+              defaultMessage="No change points found"
+            />
+          }
+          iconType="search"
+        >
+          <p>
+            <FormattedMessage
+              id="xpack.aiops.changePointDetection.noChangePointsFoundMessage"
+              defaultMessage="Try to extend the time range or update the query"
+            />
+          </p>
+        </EuiCallOut>
+      ) : null}
+
+      <EuiFlexGrid columns={annotations.length >= 2 ? 2 : 1} responsive gutterSize={'s'}>
+        {annotations.map((v) => {
+          return (
+            <EuiFlexItem key={v.group_field}>
+              <EuiPanel paddingSize="s" hasBorder hasShadow={false}>
+                <EuiFlexGroup justifyContent={'spaceBetween'} alignItems={'center'}>
+                  <EuiFlexItem grow={false}>
+                    <EuiFlexGroup alignItems={'center'} gutterSize={'s'}>
+                      <EuiFlexItem grow={false}>
+                        <EuiTitle size="xxs">
+                          <h3>{v.group_field}</h3>
+                        </EuiTitle>
+                      </EuiFlexItem>
+                      {v.reason ? (
+                        <EuiFlexItem grow={false}>
+                          <EuiToolTip position="top" content={v.reason}>
+                            <EuiIcon
+                              tabIndex={0}
+                              color={'warning'}
+                              type="alert"
+                              title={i18n.translate(
+                                'xpack.aiops.changePointDetection.notResultsWarning',
+                                {
+                                  defaultMessage: 'No change point agg results warning',
+                                }
+                              )}
+                            />
+                          </EuiToolTip>
+                        </EuiFlexItem>
+                      ) : null}
+                    </EuiFlexGroup>
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <EuiBadge color="hollow">{v.type}</EuiBadge>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+
+                {v.p_value !== undefined ? (
+                  <EuiDescriptionList
+                    type="inline"
+                    listItems={[{ title: 'p_value', description: v.p_value }]}
```

Comment:
The Log Rate Spikes page uses a hyphen in `p-value` rather than an underscore. Also the precision for the value should be reduced I think to be consistent with other pages, where 3 sigfigs is used:

<img width="207" alt="image" src="https://user-images.githubusercontent.com/7405507/199241941-af3209a2-6c2b-4043-8b12-b054df4517c8.png">

Reply:
Updated in https://github.com/elastic/kibana/pull/144093/commits/e45965162a5a5775f1d07e9334e218e402068b04

Reply:
As discussed, I now see `NaN` for quite a lot of series:

<img width="353" alt="image" src="https://user-images.githubusercontent.com/7405507/201886989-636c33d2-ec42-4329-87ce-9fa073bb1abf.png">

Reply:
fixed in https://github.com/elastic/kibana/pull/144093/commits/50979303ab2f771cc915490b3065d4a45fbb09a4